medusa/providers/torrent/html/bjshare.py

```diff
@@ -0,0 +1,250 @@
+                search_str = re.sub(r"\((\d{4})\)$", '', search_str).strip()
+
+                search_params['searchstr'] = search_str
+
+                next_page = 1
+                has_next_page = True
+                while has_next_page and next_page <= self.max_back_pages:
+
+                    search_params["page"] = next_page
+                    log.debug(u"Page Search: {0}".format(next_page))
+                    next_page += 1
+
+                    response = self.session.get(self.urls['search'], params=search_params)
+                    if not response:
+                        log.debug("No data returned from provider")
+                        continue
+
+                    result = self._parse(response.content, mode, search_string)
+                    has_next_page = result['has_next_page']
+                    items += result['items']
+
+                # For each search mode sort all the items by seeders if available
+                items.sort(key=lambda d: try_int(d.get('seeders', 0)), reverse=True)
+
+                results += items
+
+        return results
+
+    def _parse(self, data, mode, search_string):
+        """
+        Parse search results for items.
+
+        :param data: The raw response from a search
+        :param mode: The current mode used to search, e.g. RSS
+        :param search_string: Original search string
+
+        :return: A KV with a list of items found and if there's an next page to search
+        """
+        items = []
+        has_next_page = False
+        with BS4Parser(data, "html5lib") as html:
+            torrent_table = html.find("table", id="torrent_table")
+            torrent_rows = torrent_table("tr") if torrent_table else []
+
+            # year_re = re.search(r"\((\d{4})\)|(\d{4})", search_string, re.I)
+
+            # ignore next page in RSS mode
+            has_next_page = mode != 'RSS' and html.find("a", attrs={"class", "pager_next"}) is not None
+            log.debug(u"More Pages? {0}".format(has_next_page))
+
+            # Continue only if at least one Release is found
+            if len(torrent_rows) < 2:
+                log.debug("Data returned from provider does not contain any torrents")
+                return {'has_next_page': has_next_page, 'items': []}
+
+            # "", "", "Name /Year", "Files", "Time", "Size", "Snatches", "Seeders", "Leechers"
+            labels = [self._process_column_header(label) for label in torrent_rows[0]("td")]
+
+            # Skip column headers
+            for result in torrent_rows[1:]:
+                cells = result("td")
+                if len(cells) < len(labels):
+                    continue
+                try:
+                    title = cells[labels.index("Nome/Ano")].find("a", dir="ltr").get_text(strip=True)
+                    # get international title if available
+                    title = re.sub(".* \[(.*?)\](.*)", r"\1\2", title)
+
+                    for serie in self.absolute_numbering:
```

Comment:
This shouldn't be a thing. Either make it work for all animes in absulute numbering or just set the show to not anime in Medusa. I'm more inclined to the latter.

Reply:
One piece is the only anime that i'm aware that is in "absolute" numbering, the problem is that they include the season (wrong season) and episode as absolute, eg: One Piece - S08E836
836 is the latest episode in absolute numbering, that is correct, but S08 is not the current season...
So for this show, i don't see a other way to make it work, setting it as not anime will not fix it i think...

Reply:
So this indexer is consistently indexing the wrong season, for one piece releases? So there are no other release groups available for one piece, which are indexed correctly?.

Reply:
So this indexer is consistently indexing the wrong season, for one piece releases?
Yes, i already complained about that and seens like they don't care to fixing it (and many other broken things)...

So there are no other release groups available for one piece, which are indexed correctly?
You mean in another indexer on in this one? 

In this indexer, it looks that it is added "automatically", so all current and new releases will be broken until they or the source from where they get that info fix it...

And currently there is not another brazillian indexer in Medusa, i have plans to add more when i have free time (B2S-Share, Manicomio Share and Speed-Share)

Reply:
ok, in that case I don't mind fixing it in this provider as an exception.
Please include in the code the reason why this exception (technical debt) is there. So that in the future, we can easily remove it, when they decide to fix it.

Reply:
Wait, you only noticed this for One piece, or multiple animes?

Reply:
@p0psicles only for One piece

Reply:
This is a guessit issue and should not be fixed on the provider side.

Reply:
If that is the case, okay.